src/Manager/SMSManager.php

```diff
@@ -33,6 +33,20 @@ public function __construct(
     ) {
     }
 
+    /** @throws \Exception */
+    public function sendAffiliationCodeSms(Beneficiaire $beneficiary): void
+    {
+        $telephone = $beneficiary->getUser()?->getTelephone();
+        $code = $this->getRandomSmallSmsCode();
+        $beneficiary->setRelayInvitationSmsCode($code);
+        $beneficiary->setRelayInvitationSmsCodeSendAt(new \DateTime('now'));
+        $this->em->flush();
```

Comment:
Should we flush here, or more try catch the doSend, and do it in the try statement ?